airbyte-container-orchestrator/src/main/java/io/airbyte/container_orchestrator/DbtJobOrchestrator.java

```diff
@@ -72,7 +72,8 @@ public Optional<String> runJob() throws Exception {
             processFactory, NormalizationRunnerFactory.create(
                 destinationLauncherConfig.getDockerImage(),
                 processFactory,
-                NormalizationRunnerFactory.NORMALIZATION_VERSION)));
+                NormalizationRunnerFactory.NORMALIZATION_VERSION,
```

Comment:
Is normalization version here still the same across the connectors? Specially since we can have different images, I think we would want to have a version per connector.

Reply:
From looking at the fields we now have in the DB, I believe this would be coming from `NORMALIZATION_TAG`

Reply:
Yes, everything is correct. The new parameter that I added for comparison to the create method already has an image and a tag in it. As I mentioned above, the next step is to remove the create method, and instead, we will directly use the new constructor for DefaultNormalizationRunner. (instead of NormalizationRunnerFactory.create(......) -> new DefaultNormalizationRunner(processFactory, normalizationImage)).